lib/parsers/pdf_parser.rb

```diff
@@ -14,62 +15,58 @@ class FormatParser::PDFParser
   # this. The only way of solving this correctly is by adding
   # different types of PDF's in the specs.
   #
-  COUNT_MARKERS = ['Count ']
-  EOF_MARKER    = '%EOF'
+  EOF_MARKER = '%%EOF'
 
   def call(io)
     io = FormatParser::IOConstraint.new(io)
 
     return unless safe_read(io, 9) =~ PDF_MARKER
 
-    attributes = scan_for_attributes(io)
+    @xref_table = FormatParser::PDFParser::XRefTable.parse(io)
```

Comment:
Since we don't have that much extra code does it make sense to have a separate module/file for this?

Reply:
It depends, is a 100 lines 'not that much extra code'? Not to sound obtuse or anything, I'm just actually curious about how that lives within WeTransfer. 😅

Reply:
Fair fair. Let's leave it the way it has been made now and debate/refactor later ;-)